itemv2.js

```diff
@@ -70,7 +89,7 @@ const BaseItem = {
       type: "integer",
       title: "The cost of a single instance of this item.",
       description:
-        "The cost is represented in the lowest denomination (eg cents). For discounts this should be negative.",
+        "The cost is represented in the lowest denomination (eg cents). For discounts this should be negative. Excludes the cost of options.",
```

Comment:
Feel it makes more sense for amount to work the same way on all - to represent the base price.

Reply:
Yeah okay that works for me.